code/game/objects/items/devices/flash.dm

```diff
@@ -125,13 +126,19 @@
 								to_chat(user, "<span class='warning'>Something seems to be blocking the flash!</span>")
 					else
 						to_chat(user, "<span class='warning'>This mind is so vacant that it is not susceptible to influence!</span>")
-	else if(issilicon(M))
+	else if(istype(M, /mob/living/silicon))
```

Comment:
issilicon is basically this check, why the change?

#define issilicon(A) istype(A, /mob/living/silicon)